cypress/e2e/title-level-request/user-can-choose-item-to-move-tlr.cy.js

```diff
@@ -0,0 +1,168 @@
+        cy.getInstanceTypes({ limit: 1 }).then((instanceTypes) => {
+          testData.instanceTypeId = instanceTypes[0].id;
+        });
+        cy.getHoldingTypes({ limit: 1 }).then((holdingTypes) => {
+          testData.holdingTypeId = holdingTypes[0].id;
+        });
+        cy.createLoanType({
+          name: `type_${getRandomPostfix()}`,
+        }).then((loanType) => {
+          testData.loanTypeId = loanType.id;
+        });
+        cy.getMaterialTypes({ limit: 1 }).then((materialTypes) => {
+          testData.materialTypeId = materialTypes.id;
+        });
+      })
+      .then(() => {
+        instanceData.itemsData = [
+          {
+            barcode: generateUniqueItemBarcodeWithShift(),
+            status: { name: 'Available' },
+            permanentLoanType: { id: testData.loanTypeId },
+            materialType: { id: testData.materialTypeId },
+          },
+          {
+            barcode: generateUniqueItemBarcodeWithShift(),
+            status: { name: 'Available' },
+            permanentLoanType: { id: testData.loanTypeId },
+            materialType: { id: testData.materialTypeId },
+          },
+          {
+            barcode: generateUniqueItemBarcodeWithShift(),
+            status: { name: 'Available' },
+            permanentLoanType: { id: testData.loanTypeId },
+            materialType: { id: testData.materialTypeId },
+          },
+        ];
+        InventoryInstances.createFolioInstanceViaApi({
+          instance: {
+            instanceTypeId: testData.instanceTypeId,
+            title: instanceData.title,
+          },
+          holdings: [
+            {
+              holdingsTypeId: testData.holdingTypeId,
+              permanentLocationId: testData.defaultLocation.id,
+            },
+          ],
+          items: instanceData.itemsData,
+        }).then((specialInstanceIds) => {
+          instanceData.instanceId = specialInstanceIds.instanceId;
+          instanceData.holdingId = specialInstanceIds.holdingIds[0].id;
+          instanceData.itemIds = specialInstanceIds.holdingIds[0].itemIds;
+        });
+        cy.wrap(instanceData.itemsData).as('itemsToMove');
+      });
+    PatronGroups.createViaApi(patronGroup.name).then((patronGroupResponse) => {
+      patronGroup.id = patronGroupResponse;
+    });
+
+    cy.createTempUser([permissions.requestsAll.gui], patronGroup.name)
+      .then((userProperties) => {
+        userData = userProperties;
+        UserEdit.addServicePointViaApi(
+          testData.userServicePoint.id,
+          userData.userId,
+          testData.userServicePoint.id
+        );
+      })
+      .then(() => {
+        TitleLevelRequests.changeTitleLevelRequestsStatus('allow');
+        Requests.createNewRequestViaApi({
+          fulfilmentPreference: 'Hold Shelf',
+          instanceId: instanceData.instanceId,
+          pickupServicePointId: testData.userServicePoint.id,
+          requestDate: new Date(),
+          requestLevel: 'Title',
```

Comment:
please add title and item level requests and add it to the constants file: cypress\support\constants.js